samples/app-anonymous-users/nodejs/api/server/index.js

```diff
@@ -0,0 +1,168 @@
+    res.json(result);
+    console.log(result);
+  }, function (err) {
+    console.log(err); // Error: "It broke"
+    res.json(err);
+  });
+});
+
+// Get user access token
+app.post('/GetLoginUserInformation', function (req, res) {
+  var tid = req.body.tid;
+  var token = req.body.token;
+  var scopes = ["https://graph.microsoft.com/User.Read", "openid"];
+
+  // Creating MSAL client
+  const msalClient = new msal.ConfidentialClientApplication({
+    auth: {
+      clientId: process.env.MicrosoftAppId,
+      clientSecret: process.env.MicrosoftAppPassword
+    }
+  });
+
+  var oboPromise = new Promise((resolve, reject) => {
+    msalClient.acquireTokenOnBehalfOf({
+      authority: `https://login.microsoftonline.com/${tid}`,
+      oboAssertion: token,
+      scopes: scopes,
+      skipCache: true
+    }).then(async result => {
+      const client = new SimpleGraphClient(result.accessToken);
+      
+      const myDetails = await client.getMeAsync();
+      var userData = {
+        details: myDetails
+      }
+      resolve(userData);
+    }).catch(error => {
+      reject({ "error": error.errorCode });
+    });
+  });
+
+  oboPromise.then(function (result) {
+    res.json(result);
+  }, function (err) {
+    console.log(err); // Error: "It broke"
+    res.json(err);
+  });
+});
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { CloudAdapter,
+  ConfigurationServiceClientCredentialFactory, createBotFrameworkAuthenticationFromConfiguration } = require('botbuilder');
+
+const { TeamsConversationBot } = require('../bots/teamsConversationBot');
+
+const credentialsFactory = new ConfigurationServiceClientCredentialFactory({
+  MicrosoftAppId: process.env.MicrosoftAppId,
+  MicrosoftAppPassword: process.env.MicrosoftAppPassword,
+  MicrosoftAppTenantId: process.env.MicrosoftAppTenantId
+});
+
+const botFrameworkAuthentication = createBotFrameworkAuthenticationFromConfiguration(null, credentialsFactory);
+
+// Create adapter.
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
+const adapter = new CloudAdapter(botFrameworkAuthentication);
+
+// Catch-all for errors.
+adapter.onTurnError = async (context, error) => {
+  // This check writes out errors to console log .vs. app insights.
+  // NOTE: In production environment, you should consider logging this to Azure
+  //       application insights. See https://aka.ms/bottelemetry for telemetry 
+  //       configuration instructions.
+
+  console.error(`\n [onTurnError] unhandled error: ${error}`);
+
+  // Send a trace activity, which will be displayed in Bot Framework Emulator
+  await context.sendTraceActivity(
+      'OnTurnError Trace',
+      `${error}`,
+      'https://www.botframework.com/schemas/error',
+      'TurnError'
+  );
+
+  // Uncomment below commented line for local debugging.
+  // await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);
```

Comment:
Check if we have added this line related, in readme for uncomment

Reply:
Fixed